### ISSUE TITLE
Dump Collection?

### ISSUE THREAD

**Issue by andyboeh** [bensuperpc/rtl9210#1]
Thanks for this - I could successfully upgrade my SSK drive to the latest firmware. The drive seems to work properly now under Linux, i.e. it does not longer get hot.

Do you also collect user submitted dumps here? If so, I can provide you with the dump of my SSK enclosure.

**Comment by bensuperpc** [comment]
Yes, i have added what I could find, any external contributions are welcome ^^

**Comment by ivan-banha** [comment]
Firstly, thank you for the instructions. I have successfully flashed firmware 1.33.44.011824 18-01-2024 on my Chieftec CEB-M2C-TL.  

I can also create a PR with a dump config file. But, I don't want to include my serial. Will it be a problem if I use random numbers instead of the real serial?

**Comment by bensuperpc** [comment]
> Firstly, thank you for the instructions. I have successfully flashed firmware 1.33.44.011824 18-01-2024 on my Chieftec CEB-M2C-TL.
> 
> I can also create a PR with a dump config file. But, I don't want to include my serial. Will it be a problem if I use random numbers instead of the real serial?

No problem ^^

**Comment by obiwan-bartek** [comment]
Thanks for this great repository. Is there a way to dump a FW from the device (not only config)?
Another question, what does it mean "golden binary" and "normal binary" used in MPTool?
